Dockerfile

```diff
@@ -82,6 +87,14 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install Criu
+ENV CRIU_VERSION 1.7
```

Comment:
I don't think its possible to get the dump out of CI, it will have to be
done in code as you suggest.

On Fri, Dec 18, 2015 at 8:08 AM Pavel Emelyanov <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/docker/docker/pull/13602#discussion_r48022789>:
>
> > @@ -82,6 +87,14 @@ RUN cd /usr/local/lvm2 \
> >  	&& make install_device-mapper
> >  # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
> >
> > +# Install Criu
> > +ENV CRIU_VERSION 1.7
>
> @boucher <https://github.com/boucher> do you have any ideas how to get
> the dump log out ot ci? If no, maybe it's worth tuning the code just to
> print the log on the screen in case of failure?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/13602/files#r48022789>.
>